setup.py

```diff
@@ -55,6 +55,13 @@
         "esprit @ git+https://github.com/CottageLabs/esprit.git@edda12177effa0945d99302f0d453b22503e335b#egg=esprit",
         "dictdiffer @ git+https://github.com/CottageLabs/dictdiffer.git@cc86c1ca1a452169b1b2e4a0cb5fc9e6125bc572#egg=dictdiffer",
         "flask-swagger @ git+https://github.com/DOAJ/flask-swagger.git@f1dbf918d9903a588eed3cce2a87eeccc9f8cc0e#egg=flask-swagger"
+
+        # pri list and automate public list
+        'gspread~=5.10.0',
+        'oauth2client~=4.1.3',
+        'pandas~=2.0.1',  # pandas lets us generate URLs for linkcheck
```

Comment:
I'm not seeing pandas imported anywhere - does it get installed under `gspread`?

Reply:
yes, it required by `gspread-dataframe`, in `extras_require.test` also using pandas, so I move it to `install_requires`  for explicit required.